syntax = "proto3";

service chatBot {
  rpc retrieveMessage(chatPayload) returns (chatResponse) {}
}

message chatPayload {
  string message = 1;
  string user_id = 2;
  string create_at = 3;
}

message chatResponse {
  string message = 1;
  string label = 2;
  float accuracy = 3;
}